values.yaml

```diff
@@ -529,6 +529,15 @@ client:
         memory: "50Mi"
         cpu: "50m"
 
+    # Optional CA certificate that will be added to the trusted system CAs.
+    # Useful if using an S3-compatible storage exposing a self-signed certificate.
+    # Example
+    #   caCert: |
+    #     -----BEGIN CERTIFICATE-----
+    #     MIIC7jCCApSgAwIBAgIRAIq2zQEVexqxvtxP6J0bXAwwCgYIKoZIzj0EAwIwgbkx
+    #     ...
```

Comment:
For [`connectInject.certs.caBundle`](https://github.com/hashicorp/consul-helm/blob/df74135dbae999add1e9d71b3d0e72e72db691f3/values.yaml#L827-L830) we are asking that ca bundle should be provided as a base64 encoded string. We're probably doing it so that we can pass it through to the webhook registration, which requires it to be base64 encoded. 

I'm wondering if there is any reason for this value to also to be base64 encoded other than consistency. I don't think so since certificate data is already base64 encoded and so there isn't any danger for weird byte data, but curious what you think.

Reply:
I don't think so either. It's more work to base64 encode and more room for error.